test/e2e/framework/pv_util.go

```diff
@@ -805,6 +888,29 @@ func deletePD(pdName string) error {
 	}
 }
 
+func deleteRegionalPD(pdName string) error {
+	if TestContext.Provider == "gce" || TestContext.Provider == "gke" {
+		gceCloud, err := GetGCECloud()
+		if err != nil {
+			return err
+		}
+
+		err = gceCloud.DeleteRegionalDisk(pdName)
+
+		if err != nil {
+			if gerr, ok := err.(*googleapi.Error); ok && len(gerr.Errors) > 0 && gerr.Errors[0].Reason == "notFound" {
```

Comment:
Theres gotta be a better way to do this. I think theres a helper function floating around somewhere

Reply:
This check is no longer necessary since DeleteDisk() and DeleteRegionalDisk() returns no error if disk is not found. Thanks for the catch!